src/test/groovy/org/ods/doc/gen/leva/doc/fixture/LevaDocDataFixture.groovy

```diff
@@ -26,6 +26,7 @@ class LevaDocDataFixture {
         data.build = buildJobParams(projectFixture)
         data.git =  buildGitData(projectFixture)
         data.openshift = [targetApiUrl:"https://openshift-sample"]
+        data.repositories = new ProjectRepositoryFixture().load()
```

Comment:
Previously we were using  getModuleData()
If we don't need it anymor,, please clean the code.

Reply:
getModuleData is still used to generate component information sent. Please notice that when documents use component information they do not use repositories information.